### ISSUE TITLE
Project dead?

### ISSUE THREAD

**Issue by NeoCro44** [markop159/KODI-Popcorn-Time#88]
Seems owner has abandoned this project
Cant start any tv shoe anymore

**Comment by giaur500** [comment]
Hi,

Indeed please confirm is this project still alive?

**Comment by NeoCro44** [comment]
I dont think its gona be updated

**Comment by giaur500** [comment]
Well 7 months no update. Shame

**Comment by markop159** [comment]
No updates because it is still working like it should except subtitles.

**Comment by giaur500** [comment]
Not really, not all movies available by mobile Popcorn Time are available via this addon. Also, series and some movies are 720p only, while on official mobile Popcorn Time 1080p is available.

**Comment by markop159** [comment]
Addon is using the same API as official [Popcorn Time](https://github.com/popcorn-official/)

**Comment by giaur500** [comment]
I told you, some movies present in Kodi Popcorn Time on Android cannot be found with your addon, I have no idea why. Also, Android app allows to choose 1080p for series, with your addon I am not able to choose version, series are always 720p

**Comment by markop159** [comment]
Ok I understood what you said. I am just stating that this addon is using official api the same as popcorn-time.sh. I am aware that there are several versions of popcorn time that are using diferent api-s. You are probably using popcorn-time.to version which is more updated but api is a little bit different and I do not have time to implement it in this addon...

**Comment by giaur500** [comment]
Well I use this app: https://github.com/popcorn-official/popcorn-android first, more movies available, second, for each movie/serie episode I am able to select quality (it's actually different torrents). With your addon, serwies are always 720p.

Do you think Android version uses some different api?

**Comment by markop159** [comment]
OK this is weird, I will try android version and check where is the problem (it should be the same API).
On addon some of the series have the option for 1080p but only a few.
For example almost all episodes of DC's Legends of tomorrow in 2nd season have the option for 1080p.

**Comment by giaur500** [comment]
markop159: on official Android app I have ability to choose 1080p for DC's Legends of tomorrow. But your addon is sticked to one torrent only (not able to choose?) and it's always 720p version.

**Comment by markop159** [comment]
Can you check in addon settings if you have set it to 720p only? And check if you have latest version installed. Latest version is 1.7.2. I can chose multiple qualities in context menu if available (1080p, 720p, 480p). To open context menu on windows hit C on keyboard or right mouse click on episode.

I tried Android version and it is using the same API.

**Comment by giaur500** [comment]
I will check and report. Kodi 16 still supported?

**Comment by markop159** [comment]
Yes Kodi 16 still supported.

**Comment by giaur500** [comment]
Ok, my version is latest (1.7.2). I will give you specific example: DC's Legends of tomorrow, SO3E18. On official android app I'm getting: https://i.imgur.com/Erugl8I.jpg

But with your addon available variants are 720p and 480p. I have set in settings to preffer 1080p.

Since you are using the same api, something is definitely wrong.

**Comment by markop159** [comment]
OK found it. You are using https://popcorn-time.to/ version and this version is using diferent api: api.apidomain.info

As I am looking at this, this is completely different api, I currently do not have time to adapt my code to this version. Maybe sometime in the future.

**Comment by markop159** [comment]
#93 opened new issue for this.

**Comment by giaur500** [comment]
Ok, thanks for your effort. I hope it will be eventually implemented. New api provides much more better sources, worth to use it. Or even use both?

**Comment by giaur500** [comment]
Not working with latest Kodi. I guess project is dead now definitely

**Comment by markop159** [comment]
@giaur500 on which device and os it is not working, I tested on couple systems and for me it is still working on Kodi 18.3:
Raspberry Pi 2 - OSMC
PC - Windows 10
Phone - Android 9

I do not have time to do any changes regarding implementation of additional APIs or any other feature, but would try to repair plugin if it was not working (I am using it almost every day).

**Comment by giaur500** [comment]
MxqPro - Coreelec 9.0.3. Playback does not start or hangs with black screen, sound ok, no video. Sometimes it hangs device

**Comment by markop159** [comment]
@giaur500 I am sorry to hear that it does not work for you, I am guessing that build you are using does not have some of codecs. I cannot try it on this device because I do not own and do not know anyone who owns it to try.

If you want to help enable [debug logging](https://github.com/markop159/KODI-Popcorn-Time/wiki/Discussions%2C-Proposals%2C-Issues-and-Bugs), test playing some movies and upload log file somewhere so I can check what is going on.

**Comment by giaur500** [comment]
It sometimes hangs when I pause and resume, sometimes it hangs and even won't start playback. There is nothing special in logs, it flloods with message:

2019-08-01 21:07:06.487 T:3569349488   DEBUG: CVideoPlayerVideo::OutputPicture - ttd:625ms pts:131.792 Clock:131.167 Level:-1 elapsed:502.353ms
2019-08-01 21:07:07.007 T:3569349488 WARNING: CRenderManager::WaitForBuffer - timeout waiting for buffer

Log taken while it hanged after resume playback. If I download torrent and play movie from file, no issues.

**Comment by markop159** [comment]
A quick search on google of "WARNING: CRenderManager::WaitForBuffer - timeout waiting for buffer" and I found out that there are other people that are having the same problem playing live video streams on CoreELEC 9.0.3. The same issue was present in some builds of libreELEC and OpenELEC, I hope that CoreELEC will be fixed in new version.

[https://discourse.coreelec.org/t/resuming-playback-audio-momentarily-muted-keyframes/3835/21](https://discourse.coreelec.org/t/resuming-playback-audio-momentarily-muted-keyframes/3835/21)
[https://github.com/OpenELEC/OpenELEC.tv/issues/3064](https://github.com/OpenELEC/OpenELEC.tv/issues/3064)

**Comment by giaur500** [comment]
That's strange, so far only Popcorn Time affected, iptv streaming and Netflix both fine
